ui/components/Workspaces/workspace-card.js

```diff
@@ -0,0 +1,306 @@
+                        background: 'transparent',
+                      },
+                      padding: 0,
+                    }}
+                    onClick={onEdit}
+                    disabled={
+                      selectedWorkspaces?.filter((id) => id == workspaceDetails.id).length === 1
+                        ? true
+                        : false
+                    }
+                  >
+                    <Edit style={{ color: 'white', margin: '0 2px' }} />
+                  </Button>
+                  <Button
+                    style={{
+                      minWidth: 'fit-content',
+                      '&.MuiButtonBase-root:hover': {
+                        background: 'transparent',
+                      },
+                      padding: 0,
+                    }}
+                    onClick={onEdit}
+                    disabled={
+                      selectedWorkspaces?.filter((id) => id == workspaceDetails.id).length === 1
+                        ? true
+                        : false
+                    }
+                  >
+                    <Delete style={{ color: 'white', margin: '0 2px' }} />
+                  </Button>
+                </Grid>
+              </Grid>
+              <Grid>
+                {workspaceDetails.description ? (
+                  <DescriptionLabel
+                    onClick={(e) => e.stopPropagation()}
+                    style={{ padding: '10px 0', color: theme.palette.secondary.white, maxHeight: '105px' }}
+                  >
+                    {workspaceDetails.description}
+                  </DescriptionLabel>
+                ) : (
+                  <EmptyDescription
+                    onClick={(e) => e.stopPropagation()}
+                    style={{ padding: '10px 0', color: `${theme.palette.secondary.white}90` }}
+                  >
+                    No description
+                  </EmptyDescription>
+                )}
+              </Grid>
+            </Grid>
+            <Grid
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                position: 'absolute',
+                bottom: '20px',
+                width: '100%',
+                color: `${theme.palette.secondary.white}99`,
+              }}
+            >
+              <Grid xs={6} style={{ textAlign: 'left' }}>
+                <DateLabel variant="span" onClick={(e) => e.stopPropagation()}>
+                  Updated At: {formattoLongDate(workspaceDetails?.updated_at)}
+                </DateLabel>
+              </Grid>
+              <Grid xs={6} style={{ textAlign: 'left' }}>
+                <DateLabel variant="span" onClick={(e) => e.stopPropagation()}>
+                  Created At: {formattoLongDate(workspaceDetails?.created_at)}
+                </DateLabel>
+              </Grid>
+            </Grid>
+          </CardWrapper>
+        }
+      />
+    </>
+  );
+};
+
```

Comment:
@senali-d can we split this code here for the back and front components separately within same file? If we could it would be much readable